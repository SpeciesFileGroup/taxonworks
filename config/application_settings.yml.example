development:
#  default_data_directory: "/home/jdoe/my-dumps"
production:
#  backup_directory: "/mnt/backup-media/tw-dumps"
  exception_notification:
    email_prefix: "[TW-Error] "
    sender_address: "\"notifier\" <notifier@example.com>"
    exception_recipients:
      - exceptions@example.com
      - root@example.com
  action_mailer_smtp_settings:
    address: "smtp.example.com"
    port: 25
    domain: "example.com"
  mail_domain: "example.com"
  action_mailer_url_host: "www.example.com"
# test:
# selenium:
#   browser: 'firefox'            # a string, one of 'firefox' or 'chrome'
#   firefox_binary_path: ''       # optionally provided when browser: 'firefox'
#   driver_path: '/usr/local/bin/chromedriver'  # optionally provided