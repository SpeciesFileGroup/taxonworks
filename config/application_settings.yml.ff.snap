test:
  selenium:
    browser: firefox
    firefox_binary_path: /snap/firefox/current/usr/lib/firefox/firefox
    driver_path: /snap/firefox/current/usr/lib/firefox/geckodriver
